content/cluster/_index.md

```diff
@@ -1,10 +1,71 @@
 ---
 title: "Cluster"
-date: 2019-06-18T16:42:20+02:00
-lastmod: 2019-06-18T16:42:20+02:00
-publishdate: 2019-06-18T16:42:20+02:00
+date: 2020-04-21T15:22:20+02:00
+lastmod: 2020-04-21T15:22:20+02:00
+publishdate: 2020-04-21T15:30:20+02:00
 draft: false
 weight: 13
 ---
 
-Nothing to see here (yet)
+# Move from control-plane cluster to a control-plane and compute nodes cluster
+
+## Create computes
+
+```		
+oc label nodes ocp4-worker-0.example.com node-role.kubernetes.io/infra="" 
+oc label nodes ocp4-worker-1.example.com node-role.kubernetes.io/infra=""
+oc get nodes
+```
+
+## Move the router to the compute nodes
+
+```	
+oc patch ingresscontroller default -n openshift-ingress-operator --type=merge --patch='{"spec":{"nodePlacement":{"nodeSelector": {"matchLabels":{"node-role.kubernetes.io/infra":""}}}}}'
+oc patch --namespace=openshift-ingress-operator --patch='{"spec": {"replicas": 2}}' --type=merge ingresscontroller/default
+oc get pods -n openshift-ingress -o wide
```

Comment:
please paste the output if you are doing 'gets'. Actually it would be nice if you can do an oc get before and after the commands, so you can see the results.

Reply:
Don't have the output anymore :(
I can PR it in the future.